lib/experimental/Navigation/Carousel/index.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import { Placeholder } from "@/lib/storybook-utils/placeholder"
+import { Meta, StoryObj } from "@storybook/react"
+import { Carousel } from "."
+
+const meta: Meta<typeof Carousel> = {
+  component: Carousel,
+  parameters: {
+    layout: "fullscreen",
+  },
+  argTypes: {
+    autoplay: { control: "boolean" },
+  },
+}
+
+export default meta
+
+type Story = StoryObj<typeof Carousel>
+
+const SLIDES = Array.from({ length: 6 }, (_, i) => (
+  <Placeholder key={i + 1} className="h-32">
+    Slide {i + 1}
+  </Placeholder>
+))
+
+export const Default: Story = {
+  decorators: [
+    (Story) => (
+      <div className="w-full p-6">
+        <Story />
+      </div>
+    ),
+  ],
+  argTypes: {
+    autoplay: { table: { disable: true } },
+  },
+  args: {
+    autoplay: false,
+    children: SLIDES,
+  },
+}
+
+export const AutoScroll: Story = {
+  decorators: [
+    (Story) => (
+      <div className="w-full p-6">
+        <Story />
+      </div>
+    ),
+  ],
+  args: {
+    autoplay: true,
+    delay: 3000,
+    children: SLIDES,
+  },
+}
+
+export const SneakPeek: Story = {
+  decorators: [
+    (Story) => (
+      <div className="w-full py-6">
+        <Story />
+      </div>
+    ),
+  ],
+  args: {
+    itemClassName: "w-11/12 basis-auto",
```

Comment:
This seems a bit hacky, we should allow to set a couple things via props:
- Whether or not we should show a little bit of the next item from next slide, this could be a boolean prop.
- Estimated number of columns for each endpoint, maybe something like:
```
{
    0: 3,
    750: 6,
    1024: 8
}
```
So that it's 3 items when size is <750px, 6 when size is 1024px, ...

Another option would be to have a simple `numberOfColumns` prop accepting a number and we have all that logic of different number of items per breakpoint outside of this component.

- `allowToGrowInItemWithSize={300}` meaning that, when specified, our slide can grow in items dynamically when a new item with 300px fits there